Added Seldon Issue NOTE to Industrial Edge (#256)
hybrid-cloud-patterns/docs: content/patterns/industrial-edge/_index.md

Reviewer: Remove the hard line-breaks

  </script>
</div>

---
**NOTE**

> Industrial Edge on OpenShift 4.12 fails CI due to Seldon issue. This only affects the 

Reviewer: ```suggestion
Industrial Edge on OpenShift Container Platform 4.12 fails CI due to Seldon issue. This only affects the 
```

Author: Thank you @abhatt-rh !

> Anomaly Detection AI/ML portion of the pattern.  The rest of the pattern functions as

Reviewer: ```suggestion
Anomaly Detection AI/ML portion of the pattern. The rest of the pattern functions as
```

> designed.  For more information on the Seldon issue please refer to https://github.com/SeldonIO/seldon-core/issues/4339.

Reviewer: ```suggestion
designed. For more information on the Seldon issue, see https://github.com/SeldonIO/seldon-core/issues/4339.
```
